examples/lfric/eg3/matrix_vector_mm_mod.F90

```diff
@@ -68,7 +68,7 @@ module matrix_vector_mm_mod
        arg_type(GH_FIELD,    GH_READ, ANY_SPACE_1),                    &
        arg_type(GH_OPERATOR, GH_READ, ANY_SPACE_1, ANY_SPACE_1)        &
        /)
-  integer :: iterates_over = CELLS
+  integer :: operates_on = CELL_COLUMN
```

Comment:
Is the name changed from `CELL` to `CELL_COLUMN` in the `use argument_mod` statement? It doesn't seem so.

Reply:
Oops, done.